app/views/layouts/nav/staff/_program_subnav.html.haml

```diff
@@ -0,0 +1,19 @@
+.navbar.navbar-default.program-subnav
+  .container-fluid
+    %ul.nav.navbar-nav
```

Comment:
Yes to the balance-scale icon on both review and program proposals since they're very similar in action.